lib/CL/pocl_cl.h

```diff
@@ -990,6 +990,9 @@ struct _cl_device_id {
   /* The program scope variable pass takes program-scope variables and replaces
      them by references into a buffer, and creates an initializer kernel. */
   cl_bool program_scope_variables_pass;
+  /* if true, pocl_llvm_build_program will ignore pocl's OpenCL headers
+   * and rely purely on Clang's OpenCL headers */
+  cl_bool disable_pocl_opencl_headers;
```

Comment:
It usually best to avoid negations in bools (I was taught so at least), so it could be better to name it use_clang_opencl_headers?

Reply:
renamed to `use_only_clang_headers` to make it clear it's explicitly only clang headers.